sdk/digitaltwins/azure-digitaltwins/azure/digitaltwins/_digitaltwins_client.py

```diff
@@ -0,0 +1,547 @@
+        # type: (List[object], **Any) -> List["models.ModelData"]
+        """Create one or more models. When any error occurs, no models are uploaded.
+        Status codes:
+        200 (OK): Success.
+        400 (Bad Request): The request is invalid.
+        409 (Conflict): One or more of the provided models already exist.
+
+        :param models: The set of models to create. Each string corresponds to exactly one model.
+        :type models: List[object]
+        :keyword str content_type: Content type to set in the HTTP header. Default to "application/json".
+        :returns: The list of ModelData
+        :rtype: List[~azure.digitaltwins.models.ModelData]
+        :raises :class: `~azure.core.exceptions.HttpResponseError`
+        """
+        return self._client.digital_twin_models.add(models, **kwargs)
+
+    def decommission_model(self, model_id, **kwargs):
+        # type: (str, List[object], **Any) -> List["models.ModelData"]
+        """Decommissions a model.
+        200 (OK): Success.
+        400 (Bad Request): The request is invalid.
+        404 (Not Found): There is no model with the provided id.
+
+        :param model_id: The id for the model. The id is globally unique and case sensitive.
+        :type model_id: str
+        :returns: None
+        :rtype: None
+        :raises :class: `~azure.core.exceptions.HttpResponseError`
+        """
+        decommission_patch = "{ 'op': 'replace', 'path': '/decommissioned', 'value': true }"
+        return self._client.digital_twin_models.update(model_id, decommission_patch , **kwargs)
+
+    def delete_model(self, model_id, **kwargs):
+        # type: (str, **Any) -> None
+        """Decommission a model using a json patch.
+        Status codes:
+        204 (No Content): Success.
+        400 (Bad Request): The request is invalid.
+        404 (Not Found): There is no model with the provided id.
+        409 (Conflict): There are dependencies on the model that prevent it from being deleted.
+
+        :param model_id: The Id of the model to decommission.
+        :type model_id: str
+        :returns: None
+        :rtype: None
+        :raises :class: `~azure.core.exceptions.HttpResponseError`
+        """
+        return self._client.digital_twin_models.delete(model_id, **kwargs)
+
+    def get_event_route(self, event_route_id, **kwargs):
+        # type: (str, **Any) -> "models.EventRoute"
+        """Get an event route.
+        Status codes:
+        200 (OK): Success.
+        404 (Not Found): There is no event route with the provided id.
+
+        :param event_route_id: The Id of the event route.
+        :type event_route_id: str
+        :returns: The EventRoute object.
+        :rtype: object
+        :raises :class: `~azure.core.exceptions.HttpResponseError`
+        """
+        return self._client.event_routes.get_by_id(event_route_id, **kwargs)
+
+    def list_event_routes(self, max_item_count=-1, **kwargs):
+        # type: (int, **Any) -> "models.EventRoute"
```

Comment:
I think return type should ItemPaged[EventRoute]

Reply:
Fixed